blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -61,6 +63,8 @@ public HttpMetricDataQueryServer() {
         router.post("/v2.0/:tenantId/views", new HttpMultiRollupsQueryHandler());
         router.get("/v2.0/:tenantId/views/histograms/:metricName", new HttpHistogramQueryHandler());
         router.get("/v2.0/:tenantId/metrics/search", new HttpMetricsIndexHandler());
+        EventModuleLoader.loadEventModule();
```

Comment:
There is no need to call `loadEventModule` here. When we try to get the instance in the next line, it will load the module if the instance is `null`.